src/common/libflux/message_proto.c

```diff
@@ -32,7 +32,7 @@ static void proto_set_u32 (uint8_t *data, int index, uint32_t val)
 void msg_proto_setup (const flux_msg_t *msg, uint8_t *data, int len)
 {
     assert (len >= PROTO_SIZE);
-    assert (msg->proto.type != FLUX_MSGTYPE_ANY);
+    assert (msg_type_is_initialized (msg));
```

Comment:
I don't see where `msg_type_is_initialized ()` is defined, should this be `msg_type_is_valid ()`? (That is what was used in `flux_msg_create()` above. Sorry if I missed something obvious

Reply:
Uh oh that was a rebase error - will fix!  (The original name of that function was `msg_type_is_initialized()` but I later changed it to `msg_type_is_valid()`)